src/kudu/consensus/consensus.proto

```diff
@@ -142,6 +142,7 @@ enum OperationType {
   // WRITE_OP = 3;
   // ALTER_SCHEMA_OP = 4;
   CHANGE_CONFIG_OP = 5;
+  MYSQL_WRITE_OP = 6;
```

Comment:
lets stay away from MYSQL in Raft layer. If we ever want to make it a more commonly available library.
WRITE_OP_2 or WRITE_OP_EXT should be good.